Userland/Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -566,4 +566,20 @@ void HTMLElement::did_receive_focus()
     browsing_context->set_cursor_position(DOM::Position::create(realm(), *this, 0));
 }
 
+String HTMLElement::access_key_label() const
+{
+    auto access_key = get_attribute(HTML::AttributeNames::accesskey);
+    if (!access_key.has_value() || access_key->is_empty())
+        return {};
```

Comment:
Add the spec steps from the algorithm in the spec above the functions as comments.

For the start of the method, something like this:

```suggestion
// Whenever an element's accesskey attribute is set, changed, or removed, the user agent must update the element's assigned access key by running the following steps:
// FIXME: We need to update the assigned access key on attribute change, set and remove.

// 1. If the element has no accesskey attribute, then skip to the fallback step below.
auto access_key = get_attribute(HTML::AttributeNames::accesskey);
if (!access_key.has_value() || access_key->is_empty()) {
    // 4. Fallback: Optionally, the user agent may assign a key combination of its choosing as the element's assigned access key and then return.
    return {};
}
```